Yafc/Windows/WelcomeScreen.cs

```diff
@@ -208,24 +208,18 @@ protected override void BuildContents(ImGui gui) {
     }
 
     private void ProjectErrorMoreInfo(ImGui gui) {
-        gui.allocator = RectAllocator.LeftAlign;
-        gui.BuildText("Check that these mods load in Factorio", TextBlockDisplayStyle.WrappedText);
-
-        string factorioLoadedPassage = "YAFC only supports loading mods that were loaded in Factorio before. If you add or remove mods or change startup settings, " +
-            "you need to load those in Factorio and then close the game because Factorio writes some files only when exiting";
-        gui.BuildText(factorioLoadedPassage, TextBlockDisplayStyle.WrappedText);
-        gui.BuildText("Check that Factorio loads mods from the same folder as YAFC", TextBlockDisplayStyle.WrappedText);
-
-        string modRemovalPassage = "If that doesn't help, try removing all the mods that are present but aren't loaded because they are disabled, " +
-            "don't have required dependencies, or (especially) have several versions";
-        gui.BuildText(modRemovalPassage, TextBlockDisplayStyle.WrappedText);
-
+        void buildWrappedText(string message) => gui.BuildText(message, TextBlockDisplayStyle.WrappedText);
+
+        buildWrappedText("Check that these mods load in Factorio");
+        buildWrappedText("YAFC only supports loading mods that were loaded in Factorio before. If you add or remove mods or change startup settings, " +
+            "you need to load those in Factorio and then close the game because Factorio writes some files only when exiting");
```

Comment:
I feel this is not correct (explanation-wise), as far as I am aware Factorio is not writing files when exiting...? In the other PR we are even encouraging people to modify mod-list.json manually (outside of Factorio).

So we might want to take a closer look to this explanation and redesign the layout for the correct text (if needed)?

_Edit_ I saw you changed the wording a little in the followup commit, but the essentials are the same (which I doubt are (still) reflecting the actual behavior of Factorio)

Reply:
Asked the community.

Reply:
then this message might be more clear `... because Factorio writes mod-list.json only when exiting`?

Reply:
Applied in https://github.com/shpaass/yafc-ce/pull/351/commits/42a0ec3b8fbf0e09776dd47eb50276cbd319782d